jupyterhub_config.py

```diff
@@ -12,12 +12,13 @@
 c.JupyterHub.log_level = 10
 
 hub_url = "http://127.0.0.1:8000"
-
 c.JupyterHub.bind_url = hub_url
-c = install_jhub_apps(c, spawner_to_subclass=SimpleLocalProcessSpawner)
-
+c.JAppsConfig.jupyterhub_config_path = "jupyterhub_config.py"
+c.JAppsConfig.conda_envs = []
```

Comment:
To confirm, this updating this config w/ conda-store created envs will be handled when we integrate with Nebari, right?

Reply:
Yep, correct.